client/layout/masterbar/style.scss

```
@@ -565,6 +570,12 @@ body.is-mobile-app-view {
 
 	.is-support-session &.is-active {
 		background: var(--studio-orange-70);
+		--color-masterbar-icon: var(--studio-gray-5) !important;
+		--color-masterbar-text: var(--studio-gray-5) !important;
+	}
```

I'm still seeing the blue on hover on an active tab, this should remove it:

```suggestion
       --color-masterbar-highlight: var(--studio-gray-5) !important;
	}
```

---

Fixed [here](https://github.com/Automattic/wp-calypso/tree/fc3a63ca7a91a2605c69ec7e71e6f507edd2af5f)